anet-dictionary.yml

```diff
@@ -302,6 +302,17 @@ fields:
         helpText: Here you can link to a related report
         types:
           - Report
+      relatedObject:
+        type: anet_object
+        label: Related object
+        helpText: Here you can link to a related object
+        types:
+          - Report
+          - Person
+          - Position
+          - Organization
+          - Location
+          - Task 
```

Comment:
Don't forget to add this same change to `testDictionaries/no-custom-fields.yml`.

Reply:
Thanks for reminding, added.  In  fact, it looks like I was the first to put the `customfields` property for the report into the test dictionary. 
Like, it'i never been done before.

Reply:
Ah, my bad, the test dictionary should be identical to the main one, with the exception that it should contain _no_ customFields at all. So in fact you shouldn't add that change to the test dictionary. Sorry about that… :blush:

Reply:
😄 Ok, taking it back.